### PR TITLE
Add levels to Cities

### PR DESCRIPTION
**Pull request by Goven112**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by Omeliukh-Nazar — APPROVED** [review]

File: EPlast/EPlast.BLL/DTO/City/CityDTO.cs [hunk 1]
```diff
@@ -26,16 +27,9 @@ public class CityDTO
         public string Description { get; set; }
 
         [Required, MaxLength(60, ErrorMessage = "Назва вулиці розташування станиці не має перевищувати 60 символів")]
-        public string Street { get; set; }
+        public string Adress { get; set; }
```
Comment (tvardero) [inline EPlast/EPlast.BLL/DTO/City/CityDTO.cs:30]: Ad**d**ress

File: EPlast/EPlast.BLL/DTO/Region/RegionFollowerDTO.cs [hunk 2]
```diff
@@ -9,10 +11,8 @@ public class RegionFollowerDTO
         public string CityDescription { get; set; }
         public string Logo { get; set; }
         public int RegionId { get; set; }
-        public string Street { get; set; }
-        public string HouseNumber { get; set; }
-        public string OfficeNumber { get; set; }
-        public string PostIndex { get; set; }
+        public string Adress { get; set; }
```
Comment (tvardero) [inline EPlast/EPlast.BLL/DTO/Region/RegionFollowerDTO.cs:14]: Ad**d**ress

File: EPlast/EPlast.DataAccess/Entities/City/City.cs [hunk 3]
```diff
@@ -23,16 +23,9 @@ public class City
         public string Description { get; set; }
 
         [Required, MaxLength(60, ErrorMessage = "Назва вулиці розташування станиці не має перевищувати 60 символів")]
-        public string Street { get; set; }
+        public string Adress { get; set; }
```
Comment (tvardero) [inline EPlast/EPlast.DataAccess/Entities/City/City.cs:26]: Ad**d**ress

File: EPlast/EPlast.DataAccess/Entities/City/CityLevel.cs [hunk 4]
```diff
@@ -0,0 +1,9 @@
+﻿namespace EPlast.DataAccess.Entities
+{
+    public enum CityLevel
+    {
```
Comment (tvardero) [inline EPlast/EPlast.DataAccess/Entities/City/CityLevel.cs:4]: Best practice is to always add zero enum value. Like `NotSpecified = 0` or `None = 0` or `Error = 0`.

Comment (Goven112) [reply EPlast/EPlast.DataAccess/Entities/City/CityLevel.cs:4]: done

**Review by tvardero — APPROVED** [review]

**Review by Serhii-Buniak — APPROVED** [review]

**Review by vsynieok — APPROVED** [review]

**Review by DimaDiD — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Pull request merged**